easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch

```diff
@@ -0,0 +1,13 @@
+# Don't use libmpc and libmpfr to find libisl
```

Comment:
And it's already cryptographically signed with my key. Even better and just mention the author.